### PR TITLE
Get Cards info (Review, Lapses, Interval, Due, Queue) in WebView

### LINKED ISSUE #4676: Feature Request: Additional card Fields
Would it be possible to add this plugin to the app?: https://ankiweb.net/shared/info/441235634

Especially "{{info::Reviews}} - the amount of reviews that you had on a card". 

Thanks in advance ! :D

### PR DESCRIPTION
**Pull request by infinyte7**
## Purpose / Description
There are Anki addon for getting cards info (Review, Lapses, Interval, Due, Queue). But on AnkiDroid that can not works so this is implementation for getting info in WebView.
 
## Fixes
Fixes #4676

## Approach
Added JavascriptInterface to pass value from Java code to WebView

## How Has This Been Tested?
Tested on Emulator
Add following to front/back side of cards to get the values (Review, Lapses, Interval, Due, Queue)
```
<script>
console.log(AnkiDroidJS.ankiGetCardReps());
console.log(AnkiDroidJS.ankiGetCardLapses());
console.log(AnkiDroidJS.ankiGetCardInterval());
console.log(AnkiDroidJS.ankiGetCardDue());
console.log(AnkiDroidJS.ankiGetCardQueue());
</script>
```
## Learning (optional, can help others)
https://developer.android.com/reference/android/webkit/WebView.html#addJavascriptInterface(java.lang.Object,%20java.lang.String)

## Checklist
_Please, go through these checks before submitting the PR._

- [ x ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ x ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ x ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ x ] You have commented your code, particularly in hard-to-understand areas
- [ x ] You have performed a self-review of your own code

### CODE REVIEW

**Comment by infinyte7** [comment]
@mikehardy @david-allison-1 
is there any improvement needed in this and my other PR?

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java [hunk 1]
```diff
@@ -3399,5 +3399,30 @@ public boolean ankiGetCardMark() {
         public int ankiGetCardFlag() {
             return mCurrentCard.getUserFlag();
         }
+
+        @JavascriptInterface
+        public int ankiGetCardReview() {
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java:3404]: This one's up for discussion, I still don't feel it's /great/, but it feels more descriptive than `Review`
```suggestion
        public int ankiGetCardReps() {
```

**Comment by Anthropos888** [comment]
Nice feature! Thanks @infinyte7 ! Would it be possible to get the card state info in the same way? https://github.com/ankidroid/Anki-Android/issues/5587

**Comment by infinyte7** [comment]
Using above the following information of ```current card ``` will be available.
```
Reps
Lapses
Interval
Due
Queue
```

**Pull request merged**